modules/rostests/kmtests/ntos_ke/KeDpc.c

```diff
@@ -81,68 +84,74 @@ START_TEST(KeDpc)
     ok_eq_uint(Dpc.Importance, DpcImportance);
     ok_eq_uint(Dpc.Number, 0);
     ok_eq_pointer(Dpc.DpcListEntry.Flink, (LIST_ENTRY *)0x5555555555555555LL);
-    ok_eq_pointer(Dpc.DpcListEntry.Blink, (LIST_ENTRY *)0x5555555555555555LL);
+    if (GetCurrentNTVersion() < _WIN32_WINNT_WIN8)
```

Comment:
On Windows 8+, what's the value set in the Blink? Some other random value?

Reply:
> Why do you want version specific checks in the first place?

It seems to me we are simply expressing different point of views:
- me: strict Windows conformance.
- you: relaxed ReactOS future transition.

>  If both are valid implementations; then this way lets us pick the better implementation without immediately having a bunch of bogus test failures.

'valid' as in:
- me: 1 given Windows version supports only 1 value.
- you: at least one Windows version supports one of the values.
In this case, 'bogus' means to me that test passes with no reminder that there is a change to make.